src/app.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const { join } = require('path');
+const express = require('express');
+
+const compression = require('compression');
+const cookieParser = require('cookie-parser');
+const router = require('./routes/router');
+const authRouter = require('./routes/authRouter');
+const { clientError, severError } = require('./controllers/errors');
+
+require('env2')('.env');
+
```

Comment:
Great job on setting up the application! To improve the code's maintainability and organization, I suggest organizing the modules in the following order:

1. Core modules: These are built-in modules in Node.js that provide basic functionality like file system operations, networking, and buffer handling. Examples include `path`, `fs`, and `http`.

2. Third-party modules: These are modules provided by external sources, such as npm packages, that can be used to provide additional functionality. Examples include `express`, `lodash`, and `nodemailer`. Be sure to research and compare different packages before choosing one to ensure it is reliable and well-maintained.

3. Your own modules: These are modules created by you or your team that can be used across the application. Consider separating the code into smaller modules to improve organization and readability.